catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorFunctionListTest.java

```diff
@@ -0,0 +1,180 @@
+				{R.string.formula_editor_function_log, R.string.formula_editor_function_log_parameter, "log"},
+				{R.string.formula_editor_function_pi, R.string.formula_editor_function_pi_parameter, "pi"},
+				{R.string.formula_editor_function_sqrt, R.string.formula_editor_function_sqrt_parameter, "sqrt"},
+				{R.string.formula_editor_function_rand, R.string.formula_editor_function_rand_parameter, "rand"},
+				{R.string.formula_editor_function_abs, R.string.formula_editor_function_abs_parameter, "abs"},
+				{R.string.formula_editor_function_round, R.string.formula_editor_function_round_parameter, "round"},
+				{R.string.formula_editor_function_mod, R.string.formula_editor_function_mod_parameter, "mod"},
+				{R.string.formula_editor_function_arcsin, R.string.formula_editor_function_arcsin_parameter, "arcsin"},
+				{R.string.formula_editor_function_arccos, R.string.formula_editor_function_arccos_parameter, "arccos"},
+				{R.string.formula_editor_function_exp, R.string.formula_editor_function_exp_parameter, "exp"},
+				{R.string.formula_editor_function_arctan, R.string.formula_editor_function_arctan_parameter, "arctan"},
+				{R.string.formula_editor_function_floor, R.string.formula_editor_function_floor_parameter, "floor"},
+				{R.string.formula_editor_function_ceil, R.string.formula_editor_function_ceil_parameter, "ceil"},
+				{R.string.formula_editor_function_max, R.string.formula_editor_function_max_parameter, "max"},
+				{R.string.formula_editor_function_min, R.string.formula_editor_function_min_parameter, "min"},
+				{R.string.formula_editor_function_length, R.string.formula_editor_function_length_parameter, "length"},
+				{R.string.formula_editor_function_letter, R.string.formula_editor_function_letter_parameter,
+						"letter"},
+				{R.string.formula_editor_function_join, R.string.formula_editor_function_join_parameter, "join"},
+				{R.string.formula_editor_function_number_of_items, R.string
+						.formula_editor_function_number_of_items_parameter, "number of items"},
+				{R.string.formula_editor_function_list_item, R.string.formula_editor_function_list_item_parameter,
+						"list item"},
+				{R.string.formula_editor_function_contains, R.string.formula_editor_function_contains_parameter,
+						"contains"}
+		});
+	}
+
+	@Parameter
+	public @StringRes int formulaEditorFunction;
+
+	@Parameter(1)
+	public @StringRes int formulaEditorFunctionParameter;
+
+	@Parameter(2)
+	public String testName;
+
+	@Before
+	public void setUp() throws Exception {
+		Script script = BrickTestUtils.createProjectAndGetStartScript("FormulaEditorFunctionListTest");
+		script.addBrick(new ChangeSizeByNBrick(0));
+		baseActivityTestRule.launchActivity(null);
+	}
+
+	@Test
+	public void testFunctionsListElements() {
+		onBrickAtPosition(0).checkShowsText(R.string.brick_when_started);
+		onBrickAtPosition(1).checkShowsText(R.string.brick_change_size_by);
+		onBrickAtPosition(1).perform(click());
```

Comment:
this works just by accident with this very brick since the click location when clicking on the brick luckily is where the editText is located.
please click on the edit text, because else this could easily be broken with layout changes...